.gitignore

```diff
@@ -1 +0,0 @@
-/.idea/
```

Comment:
https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore you may use this git ignore , because your MR contains a lot of the garbage like .xml files. git ignore file must be placed in the root folder not in idea folder.

Reply:
Fixed
Add gitignore to project
I analyze gitignore from your link and delete from git directory out and .idea